src/commandProvider.ts

```diff
@@ -480,4 +482,99 @@ for (i = 0; i < acc.length; i++) {
   });
 }
 </script>
-`
\ No newline at end of file
+`
+
+export async function openSigconverter() {
+    let backend :string = vscode.workspace.getConfiguration("sigma").get("sigconverterBackend") || "splunk"
+    
+    let editor = vscode.window.activeTextEditor;
+    if (!editor) {
+        return;
+    }
+    let html = `<!DOCTYPE html>
+    ${SIGMACONVERTERHEAD}
+    <body height="100vh">
+        <code id="query-code" class="text-sm language-splunk-spl">
+        Loading...
+        </code>
+    </body>
+    </html>
+`
+
+let webviewPanel = vscode.window.createWebviewPanel("panel", "sigconverter.io", vscode.ViewColumn.Beside, {
+    enableScripts: true,
+        });
+    //webviewPanel.webview.html = generateWebviewContent(rule64, backend);
+    webviewPanel.webview.html = html
+
+    const updateSigconverter = () =>{
+        if (editor) {
+            let rule = editor?.document.getText();
+            translateRule(rule, backend).then((res: string) => {
+                let html = `<!DOCTYPE html>
+                                ${SIGMACONVERTERHEAD}
+                            <body height="100vh">
+                            <h3>
+                            <span>Backend:</span> <span class="text-sigma-blue">${backend}</span>
```

Comment:
Just to double check. This value can only take one of the defined ones in the enum?

I hope the `vscode.workspace.getConfiguration("sigma").get("sigconverterBackend")` is smart to return something if its not part of the defined enum x)

Reply:
null would be resolved to "splunk" a bit earlier, but just to be sure we should clean this variable aswell... thanks!